.teamcity/common/performance-test-extensions.kt

```diff
@@ -55,4 +55,6 @@ val individualPerformanceTestArtifactRules = """
         subprojects/*/build/test-results-*.zip => results
         subprojects/*/build/tmp/**/log.txt => failure-logs
         subprojects/*/build/tmp/**/profile.log => failure-logs
+        subprojects/*/build/report-*-performance-tests.zip => .
+        subprojects/*/build/performance-tests/report => report 
```

Comment:
@wolfs Do you know why this is not working https://builds.gradle.org/viewLog.html?buildId=25795786&buildTypeId=Gradle_Util_Performance_AdHocPerformanceScenarioLinux&tab=buildLog&branch_Gradle_Util_Performance=blindpirate%2Frefactor-per-test&_focus=5493#_state=5481,5488 ?

Reply:
No idea. Is the report generated locally?

Reply:
Yes:

```
# zhb @ bs-MacBook-Pro in ~/Projects/gradle on git:blindpirate/refactor-per-test o [14:46:33] 
$ ls subprojects/performance/build/performance-tests/report
css        index.html js         tests
```